xbmc/windows/GUIMediaWindow.cpp

```diff
@@ -1593,8 +1593,7 @@ bool CGUIMediaWindow::OnContextButton(int itemNumber, CONTEXT_BUTTON button)
   case CONTEXT_BUTTON_BROWSE_INTO:
     {
       CFileItemPtr item = m_vecItems->Get(itemNumber);
-      if(Update(item->GetPath()))
-        return true;
+      Update(item->GetPath());
```

Comment:
No it's good to go as is, just curious if this was originally a bug or just weird :)